packages/auth/amplify_auth_cognito/pubspec.yaml

```diff
@@ -12,6 +12,7 @@ dependencies:
   amplify_auth_cognito_ios: 0.5.0
   amplify_auth_plugin_interface: 0.5.0
   amplify_core: 0.5.0
+  amplify_flutter: 0.5.0
```

Comment:
Can we avoid this? I think it's preferable to avoid circular dependencies.

Reply:
amplify_flutter doesn't depend on amplify_auth_cognito, not sure what circular dependency you mean

Reply:
I think I may have been thinking about the old interface... this is fine.